game/ui/base/transactionscreen.cpp

```diff
@@ -386,18 +386,27 @@ void TransactionScreen::populateControlsAlien()
 {
 	int leftIndex = getLeftIndex();
 	int rightIndex = getRightIndex();
+
 	for (auto &ae : state->agent_equipment)
 	{
+		auto &alienTypeName = ae.first;
+
 		if (!ae.second->bioStorage)
 		{
 			continue;
 		}
 		// Add alien
 		for (auto &b : state->player_bases)
 		{
-			// Removing brainsucker pod from alien containment list
+			auto alienTypeExistingControl = findControlById(type, alienTypeName);
```

Comment:
Can be const?

Reply:
Even if the variable will have a new value for every loop run, is still advised to create it as const?

Reply:
If it's not changed after declaration, I'd mark it "const".

I generally see that const *should* be the default, as it means you know at a glance it won't change value later in that block.

Reply:
Updated to const:

https://github.com/OpenApoc/OpenApoc/blob/12f960b12bf8b7d40fd104f027fedb298ccdcd30/game/ui/base/transactionscreen.cpp#L401